tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -0,0 +1,123 @@
+"""Define settings classes available for environments deployed in Cloud.gov."""
+
+import json
+import os
+
+from tdpservice.settings.common import Common
+
+
+def get_json_env_var(variable_name):
+    """Retrieve and serialize a JSON environment variable."""
+    return json.loads(
+        os.getenv(variable_name, '{}')
+    )
+
+
+def get_cloudgov_service_creds_by_instance_name(services, instance_name):
+    """Retrieve credentials for a bound Cloud.gov service by instance name."""
+    return next(
+        (service.get('credentials', {}) for service in services
+         if service.get('instance_name') == instance_name),
+        {}
```

Comment:
Might be worth an extra comment explaining why next has this empty dict here. Correct me if I'm wrong, but it is for error-handling on event of an empty/exhausted iterator?

Reply:
You're correct, its to prevent errors in the case there is no matching service for the given instance name. Although I'm considering updating it to return a [defaultdict](https://docs.python.org/3/library/collections.html#collections.defaultdict) instead to prevent KeyErrors down the line.